i18n/extract.py

```diff
@@ -179,6 +179,16 @@ def run(self, args):
         self.rename_source_file('django-saved.po', 'django.po')
         self.rename_source_file('djangojs-saved.po', 'djangojs.po')
 
+        # Remove the POT-Creation-Date from the all files.
+        for filename in files_to_clean.union({
+            'django.po',
+            'djangojs.po',
+        }):
```

Comment:
nitpick: `union` sounds cryptic to me, I recommend using something more readable like:

```suggestion
        all_files_to_clean = files_to_clean.union({'django.po', 'djangojs.po'})
        # Remove the POT-Creation-Date from the all files.
        for filename in all_files_to_clean:
```



BTW, `union` is similar to `|`, but I'm unsure which is more readable than the other

```py
>>> set([1, 2]).union(set([2, 3]))
{1, 2, 3}
>>> set([1, 2]) | set([2, 3])
{1, 2, 3}
```

Reply:
Done. And I always see `union` as more readable than `|` even if the latter is more Pythonic